internal/sync/configs/frrk8-config.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
You don't need to create new file. Simple add these lines to ./internal/sync/configs/metallb-config.yaml

Reply:
removed frr file and added to metallb